.ddev/config.yaml

```diff
@@ -0,0 +1,173 @@
+name: tea
+type: php
+docroot: ""
+php_version: "7.4"
+webserver_type: nginx-fpm
+router_http_port: "80"
+router_https_port: "443"
```

Comment:
These ports will not work on development machines that already have a local web server (i.e., also my computer). We should use different local ports for this.

On the other hand, we do not really need a webserver for developing the extension as we only use the containered PHP for running code checks and for running the unit tests. Can we disabled the web server in ddev?

Reply:
Yea, we can change it to other port, but, if I work only on ddev, then there is no problem in running multiple instances of docker on port 80 and 443. 

I don't think that it's possible to disable web container entirely, as it's PHP container at the same time.

Reply:
In his ddev setup, Helmut disabled some containers:
https://github.com/helhum/TYPO3-Distribution/blob/master/.ddev/config.yaml#L20

Maybe this can help us here, too?

Reply:
I disabled phpmyadmin and ssh agent